js/graviex.js

```diff
@@ -0,0 +1,1012 @@
+        //     "base_fixed": 4,
+        //     "base_fee": 0.002,
+        //     "quote_unit": "btc",
+        //     "quote_fixed": 8,
+        //     "quote_fee": 0.002,
+        //     "api": true,
+        //     "base_lot": null,
+        //     "quote_lot": null,
+        //     "base_min": "0.00000010",
+        //     "quote_min": "0.00000010",
+        //     "blocks": 260722,
+        //     "block_time": "2022-01-31 14:10:08",
+        //     "wstatus": "on",
+        //     "low": "0.0000005",
+        //     "high": "0.00000058",
+        //     "last": "0.00000058",
+        //     "open": "0.000000510",
+        //     "volume": "70673.0412",
+        //     "volume2": "0.03837782973",
+        //     "sell": "0.00000058",
+        //     "buy": "0.00000055",
+        //     "at": 1643628427,
+        // },
+        if (Array.isArray (response)) {
+            const firstTicker = this.safeValue (response, 0, {});
+            return this.parseTicker (firstTicker, market);
+        }
+        return this.parseTicker (response, market);
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const response = await this.publicGetTickers (params);
+        // {
+        //   "giobtc": {
+        //     "name": "GIO/BTC",
+        //     "base_unit": "gio",
+        //     "base_fixed": 4,
+        //     "base_fee": 0.002,
+        //     "quote_unit": "btc",
+        //     "quote_fixed": 8,
+        //     "quote_fee": 0.002,
+        //     "api": true,
+        //     "base_lot": null,
+        //     "quote_lot": null,
+        //     "base_min": "0.00000010",
+        //     "quote_min": "0.00000010",
+        //     "blocks": 260722,
+        //     "block_time": "2022-01-31 14:10:08",
+        //     "wstatus": "on",
+        //     "low": "0.0000005",
+        //     "high": "0.00000058",
+        //     "last": "0.00000058",
+        //     "open": "0.000000510",
+        //     "volume": "70673.0412",
+        //     "volume2": "0.03837782973",
+        //     "sell": "0.00000057",
+        //     "buy": "0.00000055",
+        //     "at": 1643628325,
+        //   },
+        // },
+        // const data = response;
+        // let timestamp = data['at'];
+        // let tickers = data['ticker'];
+        const ids = Object.keys (response);
+        const result = {};
+        for (let i = 0; i < ids.length; i++) {
+            const id = ids[i];
+            const market = this.safeMarket (id);
+            const symbol = market['symbol'];
+            result[symbol] = this.parseTicker (response[id], market);
+        }
+        return result;
+    }
+
+    parseTrade (trade, market = undefined) {
+        const price = this.safeString2 (trade, 'p', 'price');
+        const amount = this.safeString2 (trade, 'volume', 'amount');
+        const funds = this.safeString (trade, 'funds');
+        const marketId = this.safeString (trade, 'symbol');
```

Comment:
I believe this should be `market` instead 
![image](https://user-images.githubusercontent.com/43336371/153564970-034dc74b-df79-46cb-9595-ae9d72e5621c.png)

Reply:
@carlosmiei, Could you please explain what exactly needs to be fixed here. As from the comment, it's not pretty clear.

Reply:
@carlosmiei  Okay thanks. Fixed in the latest commit https://github.com/ccxt/ccxt/pull/11697/commits/bb2b0b2ef29c557eee9dad97fee9c010de932c9a